docs/src/app/(public)/(content)/react/components/combobox/demos/decorated-input/tailwind/index.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from 'react';
+import { Combobox } from '@base-ui-components/react/combobox';
+
+export default function ExampleInputInsideTrigger() {
+  const id = React.useId();
+  const [inputValue, setInputValue] = React.useState('');
+  const isTyping = inputValue.trim() !== '';
+
+  return (
+    <Combobox.Root
+      items={people}
+      itemToStringLabel={(p) => p.name}
+      inputValue={inputValue}
+      onInputValueChange={(next, eventDetails) => {
+        if (eventDetails.reason === 'item-press') {
+          // Keep the input blank after selection for overlay display.
+          setInputValue('');
+          return;
+        }
+
+        setInputValue(next);
+      }}
+    >
+      <div className="flex flex-col gap-1 text-sm leading-5 font-medium text-gray-900">
+        <label htmlFor={id}>Assignee</label>
+        <Combobox.Value>
+          {(person: Person | null) => (
+            <div
+              className="relative group flex w-[16rem] min-[500px]:w-[20rem]"
+              aria-label={person ? person.name : undefined}
+            >
+              <Combobox.Input
+                id={id}
+                placeholder={person ? '' : 'Select person'}
+                className="h-10 w-full rounded-md font-normal border border-gray-200 pl-9 pr-16 text-base text-gray-900 bg-[canvas] group-hover:bg-gray-100 group-focus-within:group-hover:bg-[canvas] focus:outline focus:outline-2 focus:-outline-offset-1 focus:outline-blue-800 cursor-default focus:cursor-text"
+                onBlur={() => {
+                  setInputValue('');
+                }}
```

Comment:
This pattern requires controlling the input value, with this special handler

Reply:
This is just how this pattern works I think 😅  

Although Vercel makes it look like a placeholder when focused. That's probably the best UX.

https://github.com/user-attachments/assets/51e79e55-403d-4fc6-860b-10cf0a78f6b7

Reply:
The main problem with that pattern is that selecting an item keeps focus on the input, so it shows the "placeholder" after selection too, which looks odd

Vercel just blurs the input and focus gets lost which is not accessible...

Reply:
> arrowing through the text doesn't work

Doesn't this negate the example? Seems like something we shouldn't present in our docs?

How would it work with `multiple`? Isn't this a use case for Chips? It strikes me as similar Gmail's "To:" UI, where you can add multiple emails and they are listed as chips with avatars.